base_station/gui/README.md

```diff
@@ -53,41 +88,139 @@ Publishers: N/A
 Usage: Displays odometry data on GUIs
 
 Sensors
-"/sensors
+"/sensors"
 File: Sensors.lcm
 Subscritions: SATask.vue, AutonTask.vue, RATask.vue
 Publishers: N/A
 Usage: 
 
 Temperature
-"/temperature
+"/temperature"
 File: Temperature.lcm
 Subscritions: SATask.vue, AutonTask.vue, RATask.vue
 Publishers: N/A
 Usage: 
 
+Xbox
+"/ra_control"
+File: Xbox.lcm
+Subscriptions: N/A
+Publishers: ArmControls.vue
+Usage: Values based on xbox controller input
+
 #### Science GUI:
+CarouselData
+"/carousel_data"
+File: CarouselData.lcm
+Subscriptions: Carousel.vue
+Publishers: N/A
+Usage: Verifies Carousel position 
+
+CarouselClosedLoopCmd
+"/carousel_closedloop_cmd"
+File: CarouselClosedLoopCmd.lcm
+Subscriptions: N/A
+Publishers: Carousel.vue
+Usage: Sends target carousel position
+
+CarouselOpenLoopCmd
+"/carousel_openloop_cmd"
+File: CarouselOpenLoopCmd.lcm
+Subscriptions: N/A
+Publishers: Carousel.vue
+Usage: Sends out throttle for Carousel control
+
+FuseData
+"/fuse_data"
+File: FuseData.lcm
+Subscriptions: PDBFuse.vue
+Publishers: N/A
+Usage: 
+
+Heater
+"/heater_state_data"
+File: Heater.lcm
+Subscriptions: Amino.vue
+Publishers: N/A
+Usage: Used to determine whether each heater is on or off
+
+Heater
+"/heater_cmd"
+File: Heater.lcm
+Subscriptions: N/A
+Publishers: Amino.vue
+Usage: Sends a message to change heater on/off
+
+HeaterAutoShutdown
+"/heater_auto_shutdown_data"
+File: HeaterAutoShutdown.lcm
+Subscriptions: Amino.vue
+Publishers: N/A
+Usage: Used to determine if heater autoshutdown is on/off
+
+HeaterAutoShutdown
+"/heater_auto_shutdown_cmd"
+File: HeaterAutoShutdown.lcm
+Subscriptions: N/A
+Publishers: Amino.vue
+Usage: Sends bool with newly selected heater autoshutdown on/off value
+
 MosfetCmd
 "/mosfet_cmd"
 File: MosfetCmd.lcm
 Subscriptions: Chrlophyll.vue
-Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue
+Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue, Raman.vue, ScoopUV.vue
```

Comment:
I don't think chlorophyll subscribes to mosfet_cmd? Or if it does we should maybe stop doing that. Also watch the spelling on chlorophyll, I know this was already here.